content/en/monitors/monitor_status.md

```diff
@@ -17,35 +17,45 @@ further_reading:
 
-Once you have [created your monitor][1], the Monitor Status page gives you a bird's eye view of your monitor status over time.
-This page is split in 3 main sections:
+* [Header](#header)
+* [Properties](#properties)
+* [Status and History](#status-and-history)
+* [Events](#events)
 
-* **[Properties](#properties)**
-* **[Status and History](#status-and-history)**
-* **[Events](#events)**
+These sections are open by default. The last three sections can be closed by using the toggle (&or;) icon to the left of the section name.
 
-These sections are open by default, and can be closed by using the little `>` icon to the left of the section name.
+## Header
+On the left, the header contains the monitor's status, time of status, and monitor title.
 
-### Monitor status options
+On the right, the header contains **Mute**, **Resolve**, and the **Settings** cog.
 
-Some options are available in the upper right corner of the page:
+### Mute
+Choose to mute a monitor directly on its status page. Use the **Scope** field to narrow your downtime. Refer to [Downtimes][2] to learn how to mute multiple scopes or multiple monitors at the same time.
 
-* **Mute**: Choose to mute a monitor directly on its status page. Use the *Scope* field to narrow your downtime.
-    Refer to the [dedicated downtime documentation][2] to learn how to mute multiple scopes or multiple monitors at the same time.
+**Note**: Muting or unmuting a monitor with the UI deletes all scheduled downtimes associated with that monitor.
 
-    NOTE: muting or unmuting a monitor via the UI deletes all scheduled downtimes associated with that monitor.
+### Resolve
+If your monitor is in an alert state, the **Resolve** button is visible. Use this button to resolve your monitor manually.
 
-    {{< img src="monitors/monitor_status/status_mute_monitor.png" alt="status mute monitor" responsive="true" style="width:30%;">}}
+The monitor `resolve` function is artificially switching the monitor status to `OK` for its next evaluation. The next monitor evaluation is performed normally on the data the monitor is based on.
 
-* **Resolve**: Resolve your monitor manually.
+If a monitor is alerting because its current data corresponds to its `ALERT` state, `resolve` has the monitor follow the state switch `ALERT -> OK -> ALERT`. Thus, it's not appropriate for acknowledging that you have seen the alert or telling Datadog to ignore the alert.
 
-* **Configuration options**: Use the *cog* icon to display options available:
+Manually `resolve`-ing a monitor is appropriate for cases where data is reported intermittently: after triggering an alert, the monitor doesn't receive further data so it can no longer evaluate alerting conditions and recover to the `OK` state. In that case the `resolve` function or the `Automatically resolve monitor after X hours` switches the monitor back to `OK` state.
```

Comment:
```suggestion
Manually `resolve`-ing a monitor is appropriate for cases where data is reported intermittently: after triggering an alert, the monitor doesn't receive further data so it can no longer evaluate alerting conditions and recover to the `OK` state. In that case, the `resolve` function or the `Automatically resolve monitor after X hours` switches the monitor back to `OK` state.
```